packages/metro-transformer/src/react-native/index.ts

```diff
@@ -0,0 +1,11 @@
+import { createLinguiMetroTransformer } from "../metroTransformer"
+
+const reactNativeTransformer = (() => {
+  try {
+    return require("@react-native/metro-babel-transformer")
+  } catch (error) {
+    return require("metro-react-native-babel-transformer")
+  }
```

Comment:
these two also should be in package json as optional peers

Reply:
done

Reply:
I ditched the `metro-react-native-babel-transformer` support because it's old and used import for `@react-native/metro-babel-transformer`